### PR TITLE
simple counters implementation

### PR DESCRIPTION
**Pull request by Marchuck**
hide drag & drop functionality

### CODE REVIEW

File: app/src/main/java/org/openhdp/hdt/ui/tracking/TrackingItem.kt [hunk 1]
```diff
@@ -4,27 +4,35 @@ import androidx.annotation.ColorInt
 
 
 data class TrackingItem(
-    val id: String,
+    val stopWatchId: String,
```
Comment (KrzysiekDev) [inline app/src/main/java/org/openhdp/hdt/ui/tracking/TrackingItem.kt:7]: I think it should be stopwatchId.

**Review by KrzysiekDev — CHANGES_REQUESTED** [review]
Looking very cool, I see only some minor problems.

File: app/src/main/java/org/openhdp/hdt/data/DataTrackingDatabase.kt [hunk 2]
```diff
@@ -8,7 +8,7 @@ import org.openhdp.hdt.data.entities.*
 @Database(
     entities = [Stopwatch::class, Category::class, Timestamp::class,
         DailyChangedDuration::class, IndependentCategories::class],
-    version = 1
+    version = 9
```
[the remark this answers is not in the document]
Comment (Marchuck) [reply app/src/main/java/org/openhdp/hdt/data/DataTrackingDatabase.kt:11]: I made a few changes in schema during testing, each new query/models change requires db version update

File: app/src/main/java/org/openhdp/hdt/ui/tracking/TrackingItem.kt (hunk 1 above)
Comment (Marchuck) [reply app/src/main/java/org/openhdp/hdt/ui/tracking/TrackingItem.kt:7]: it's UI model, I promise to change it in upcoming PR 🤞

File: app/src/main/java/org/openhdp/hdt/data/dao/TimestampDAO.kt [hunk 3]
```diff
@@ -1,20 +1,26 @@
...
-    @Query("SELECT * FROM timestamps")
-    fun getAllTimestamps(): LiveData<List<Timestamp>>
+    @Query("SELECT COUNT() FROM timestamps")
+    suspend fun getAllTimestampsCount(): Int
+
+    @Query("SELECT * FROM timestamps WHERE stopwatchId=:id ORDER BY startTime DESC LIMIT 1")
+    suspend fun lastTimestampOf(id: String): Timestamp?
+
+    @Query("SELECT * FROM timestamps WHERE stopwatchId=:id ORDER BY startTime ASC")
+    suspend fun getTimestampsFrom(id: String): List<Timestamp>
```
[the remark this answers is not in the document]
Comment (Marchuck) [reply app/src/main/java/org/openhdp/hdt/data/dao/TimestampDAO.kt:25]: Live data does not saves the day, I need also handle edge cases, errors & empty lists

**Review by KrzysiekDev — APPROVED** [review]
OK, we can merge this PR.

**Pull request merged**